### PR TITLE
Enable string functions on marcosschriekwiki for T812

### PR DESCRIPTION
**Pull request by LegoFan4000**

### CODE REVIEW

**Comment by Reception123** [comment]
This will be on hold ATM, per @NDKilla 's comment on https://phabricator.miraheze.org/T812 .

**Comment by Reception123** [comment]
I've checked mediawiki.org and this would be integrated into ParserFunction. Will still wait for another sysadmins approval before merging.

**Comment by LegoFan4000** [comment]
@Reception123 that what this is for.

**Comment by JohnFLewis** [comment]
Why is this adding an undefined wmg variable?

**Comment by Reception123** [comment]
(I might be wrong but) I think its supposed to be defined in LocalExtensions.php and then used as a wmg.

**Comment by Reception123** [comment]
It's not done like that. You would need to define it under the extension, so therefore this could actually go to GlobalExtensions (but again not sure this is the correct approach)

File: LocalSettings.php [hunk 1]
```diff
@@ -3746,4 +3751,3 @@ function onSiteNoticeAfter( &$siteNotice, $skin ) {
 
 	return true;
 } */
-
```
Comment (Reception123) [inline LocalSettings.php]: please add this line back

**Comment by JohnFLewis** [comment]
Why are all these changes being made? wg in LocalSettings was correct as this is a global extension!

**Comment by JohnFLewis** [comment]
Please make a new request. The amount of changes to this is getting way too much.

**Comment by Reception123** [comment]
@LegoFan4000 Sorry, but I did tell you that I wasn't sure, you should have waited for a confirmation.

**Pull request closed without merging**